### DISCUSSION TITLE
Request to Create a Custom Chinese Localization Patch

### DISCUSSION THREAD

**Discussion by Dianli-Moli** [flybywiresim/aircraft#8820] [General]
### Aircraft Version

Stable

### Description

I want to localize the checklist, instrument panel, buttons, etc., for the A32NX into Chinese, but I’m not sure how to complete this step. Could someone knowledgeable please point me to the file location and method for doing the localization adaptation?

**Comment by alepouna** [comment]
Hi, thank you for the interest. I have went ahead and converted this to a discussion rather than an bug report. 

Please note that you can not alter the instruments language and I'm fairly sure not the buttons and panels either as that requires modifications to the model (modelers/textures correct me if wrong). 

The currently available translations are for the EFB, you can find the process here: https://github.com/flybywiresim/aircraft/blob/master/fbw-a32nx/src/localization/README.md
